docs/api/ref/schemas/product_hidden.yaml

```diff
@@ -0,0 +1,117 @@
+type: object
+detail: |
+  Here are referenced fields that one may found in the database but
+  either have no real meaning or should not be considered part of the API
+
+properties:
+  allergens_imported:
+    type: string
+  brands_imported:
+    type: string
+  countries_imported:
+    type: string
+  data_sources_imported:
+    type: string
+  lang_imported:
+    type: string
+  lc_imported:
+    type: string
+  no_nutrition_data_imported:
+    type: string
+  nutrition_data_per_imported:
+    type: string
+  obsolete_imported:
+    type: string
+  owner_imported:
+    type: string
+  packaging_imported:
+    type: string
+  producer_version_id_imported:
+    type: string
+  quantity_imported:
+    type: string
+  serving_size_imported:
+    type: string
+
+
+  grades:
+    type: object
+
+  countries_beforescanbot:
+    type: string
+  nucleotides_prev_tags:
+    type: array
+    items:
+      type: object
+
+  nutrition_data:
+    type: string
+  nutrition_data_per:
+    type: string
+  nutrition_data_prepared:
+    type: string
+  nutrition_data_prepared_per:
+    type: string
+ 
+
+  _id:
+    type: string
+    description: id in database of the product, this normally is the barcode
+  _keywords:
+    type: array
+    items:
+      type: string
+
+  max_imgid:
+    type: string
+  packaging:
+    type: string
+  packaging_hierarchy:
+    type: array
+    items:
+      type: string
+  packaging_lc:
+    type: string
+  packaging_tags:
+    type: array
+    items:
+      type: string
+  producer_version_id:
+    type: string
+  removed_countries_tags:
+    type: array
+    items:
+      type: object
+  sortkey:
+    type: integer
+
+  states:
+    type: string
+  states_hierarchy:
+    type: array
+    items:
+      type: string
+  states_tags:
```

Comment:
states_tags probably should not be hidden, it's what we have in /states and that is shown at the bottom of product pages

Reply:
Moved to product_quality